batcher/aligned-batcher/src/lib.rs

```diff
@@ -449,15 +449,15 @@ impl Batcher {
         // Set the batch posting flag to true
         *batch_posting = true;
 
-        let current_batch_size = serde_json::to_vec(&batch_verification_data).unwrap().len();
+        let current_batch_size = cbor_serialize(&batch_verification_data).unwrap().len();
```

Comment:
this may be out of the scope of this particular PR, but lets handle this unwraps. If there is a serialization error, lets send the error message to the client and return Ok() here.

Reply:
Fixed [f0f5354](https://github.com/yetanotherco/aligned_layer/pull/746/commits/f0f53540a95ec50c447ed30c61ff3eefb02c3701)